lib/entry.ts

```diff
@@ -140,13 +140,26 @@ type IDBPDatabaseExtends = Omit<
   'createObjectStore' | 'deleteObjectStore' | 'transaction' | 'objectStoreNames'
 >;
 
+/**
+ * A variation of DOMStringList with precise string types
+ */
+export interface TypedDOMStringList<
+  T extends string
+> extends DOMStringList {
+    readonly length: number;
```

Comment:
Doesn't it already have this by extending `DOMStringList`?

Reply:
Yup, oops